Task 5/middlewares/passportFacebookMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import passport from 'passport';
+import FacebookStrategy from 'passport-facebook';
+import _ from 'underscore';
+
+const userData = [{facebookId: "1",
+               firstName: "Yahor",
+               lastName: "Pamakha",
+               email: "email@google.com",
+               isActive: "true"
+              }];
+
+export default passport.use(new FacebookStrategy({
```

Comment:
There's no need to export the applied  strategy.
Just use passport.authenticate(<strategy naming according to documentation>) as a middleware for your routes.